cms/src/test/java/uk/nhs/digital/apispecs/ApiSpecificationPublicationServiceTest.java

```diff
@@ -325,9 +325,9 @@ public void sync_handlesMultipleSpecifications() {
 
         // published before but not changed in Apigee - expect to not be updated nor published
         final String publishedNotChanged_id                = "publishedNotChanged";
-        final String publishedNotChangedRemote_modTime     = "2020-05-10T10:30:00.001Z";
-        final String publishedNotChangedOld_checkTime      = "2020-05-10T10:30:00.002Z";
-        final Instant publishedNotChanged_newCheckTime     = Instant.parse("2020-05-10T10:30:00.002Z");
```

Comment:
not sure why these changes have been made?

Reply:
The reason for shifting a few string date values to the right was to ensure they align with the values that are preceded by `Instant.parse`, so that it's easier to see the differences between them when all the components of the dates are in the same column (values that are close together only differ in single component). The test values in this method are difficult to grasp as they are, so this is my attempt to make it easier, if only by a smidgen.

The changes within the values themselves was to move the differences from milliseconds to seconds, to ensure that minuscule timing variations in tests don't produce varying results. Admittedly, this particular change does not appear to be needed. The reason I put it in place was because there is a weird difference in how IDE runs the tests vs Maven. This particular test fails in my IDE (but only when run in a larger group of tests) but passes in Maven; fixing it for IDE makes it fail in Maven. The too-small-differences in the time values were my initial suspected culprit so I increased them from milliseconds to seconds. At the end, I didn't resolve the problem (the test passes in Maven which is what we care about more than IDE) but left the changes in place.

Happy to revert the latter change, if you like?

Reply:
sure, i don't have a problem with the latter change but if it's not fixing the problem behaviour you're seeing, may as well revert it :)

Reply:
Good point; the changes to values have now been reverted.